scrapyd_k8s/launcher/docker.py

```diff
@@ -1,9 +1,9 @@
 import re
-import os
 import socket
 
 import docker
-from ..utils import native_stringify_dict
+
+from scrapyd_k8s.utils import format_iso_date_string, native_stringify_dict
```

Comment:
I'm not fully sure if moving to absolute imports has unintended consequences.

Reply:
e.g. running the daemon from a different PWD (not sure if that works anyway), it might e.g. impact the Docker image